src/smt/command.cpp

```diff
@@ -430,6 +492,23 @@ Command* CheckSatCommand::clone() const
 
 std::string CheckSatCommand::getCommandName() const { return "check-sat"; }
 
+void CheckSatCommand::toStream(std::ostream& out,
+                               int toDepth,
+                               bool types,
+                               size_t dag,
+                               OutputLanguage language) const
+{
+  if (d_expr.isNull())
```

Comment:
I think I'd prefer to make `Printer::toStreamCmdCheckSat` robust to whether the expression was null and avoid the case split here.